vimeo-networking/src/main/java/com/vimeo/networking/utils/PrivacySettingsParams.java

```diff
@@ -0,0 +1,91 @@
+package com.vimeo.networking.utils;
+
+import com.vimeo.networking.Vimeo;
+import com.vimeo.networking.model.Privacy;
+import com.vimeo.networking.model.Privacy.PrivacyCommentValue;
+import com.vimeo.networking.model.Privacy.PrivacyEmbedValue;
+import com.vimeo.networking.model.Privacy.PrivacyViewValue;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Builder for setting privacy params for a video.
+ *
+ * Created by Mohit Sarveiya on 3/20/18.
+ */
+public final class PrivacySettingsParams {
```

Comment:
Considering that this class creates a map that is not type safe, it would be good to have some unit tests that verify that the map contains the expected values.